code/style.css

```diff
@@ -1,4 +1,160 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: sans-serif;
+}
+
+.body-container {
+width: 1400px;
+height: 100%;
+}
+
+.header-container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.right-header {
+  display: none;
+}
+
+.main-news {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 1em 0 1em 0;
+}
+
+.left-main img {
+  width: 100%;
+  height: 100%;
+}
+
+.left-header {
+  display: flex;
+  justify-content: center;
+}
+
+.right-main p{
+  font-size: 12px;
+}
+
+.small-news {
+  grid-gap: 1em;
+  display: grid;
+  grid-template-columns: 1fr;
+  }
+
+.small-box{
+  position: relative;
+}
+
+.image {
+  opacity: 1;
+  display: block;
+  width: 100%;
+  height: auto;
+  transition: .5s ease;
+  backface-visibility: hidden;
+}
+
+.middle {
+  transition: .2s ease;
+  opacity: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.small-box:hover .image {
+  opacity: 0.3;
+}
+
+.small-box:hover .middle {
+  opacity: 1;
+}
+
+.text {
+  color: black;
+  font-size: 16px;
+  padding: 16px 32px;
+}
+
+
+.small-box, .left-main {
+box-shadow: 2.2px 4.5px 4.5px rgba(122, 115, 96, 0.43);
+}
+
+.small-box img {
+  width: 100%;
+  height: 100%;
+  grid-gap: 1em;
+}
+
+
+
+
+@media (min-width: 668px) {
+  .header-container {
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 1em;
+  }
+  
+  .left-header {
+    justify-content:flex-start;
+  }
+
+  .left-header img {
+  height: 300px;
+  width: 300px;
+  }
+
+  .right-header {
+    display:block;
+  }
+
+  .header-links {
+    display: flex;
+    justify-content:space-between;
+    list-style: none;
+    cursor: pointer;
+    margin-top:130px;
+    }
```

Comment:
To get the header links in the middle instead of at the far end of the right side try to decide a width for the link box.

Reply:
Thank you Lisa, I actually wanted the links to be on the right side. What I did not manage to do was to a hoover effect on them though, tips are welcome for next project :)